ajax/libs/jquery-ui-multiselect-widget/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "jquery-ui-multiselect-widget",
+  "filename": "jquery.multiselect.min.js",
+  "description": "MultiSelect progessively enhances an ordinary multiple select control into elegant drop down list of checkboxes, stylable with ThemeRoller.",
+  "version": "1.17",
+  "keywords": [
+    "jquery",
+    "jquery ui",
+    "ThemeRoller",
+    "multiple select"
+  ],
+  "author": {
+    "name": "Eric Hynds",
+    "url": "https://github.com/ehynds"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/ehynds/jquery-ui-multiselect-widget"
+  },
+  "licenses": [
+    "MIT",
+    "GPL-2.0"
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/ehynds/jquery-ui-multiselect-widget.git",
+    "fileMap": [
+      {
+        "basePath": "",
```

Comment:
I can't make it into one line.
I tried 
`"?(i18n)/jquery.multiselect*"`
`"@(i18n|.)/jquery.multiselect*"`
`"@(i18n/|./)jquery.multiselect*"`
`"@(jquery.multiselect*|i18n/jquery.multiselect*)"`
It would be nice to have some hint. Thanks.